Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35473&attempt_id=cff221b0-b591-4249-a51a-828d05e89acd)

<details><summary>Partial log (click to expand)</summary><p>

```
make[3]: Entering directory '/private/tmp/nix-build-valentina-0.6.0.0a.drv-0/hg-archive/src/app/tape'
make[3]: *** No rule to make target '/nix/store/7z59q30va8aaansxh7bv74hcsl3csnl9-qtbase-5.10.1/translations/qtbase_ru.qm', needed by 'bin/tape.app/Contents/Resources/translations/ru_RU.lproj/qtbase_ru.qm'.  Stop.
make[3]: Leaving directory '/private/tmp/nix-build-valentina-0.6.0.0a.drv-0/hg-archive/src/app/tape'
make[2]: *** [Makefile:46: sub-tape-make_first] Error 2
make[2]: Leaving directory '/private/tmp/nix-build-valentina-0.6.0.0a.drv-0/hg-archive/src/app'
make[1]: *** [Makefile:71: sub-app-make_first] Error 2
make[1]: Leaving directory '/private/tmp/nix-build-valentina-0.6.0.0a.drv-0/hg-archive/src'
make: *** [Makefile:45: sub-src-make_first] Error 2
builder for '/nix/store/f2rzjyg59lwk6aq1n46xpkf8dii58ciz-valentina-0.6.0.0a.drv' failed with exit code 2
error: build of '/nix/store/f2rzjyg59lwk6aq1n46xpkf8dii58ciz-valentina-0.6.0.0a.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/valentina/default.nix
+++ b/pkgs/applications/misc/valentina/default.nix
@@ -1,0 +1,67 @@
+{ stdenv, fetchhg
+, qmake, qttools
+, qtbase, qtsvg, qtxmlpatterns
+, poppler_utils
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "valentina-${version}";
+  version = "0.6.0.0a";
+
+  src = fetchhg {
+    url = "https://bitbucket.org/dismine/valentina";
+    rev = "ccd68eba533a82aeb2dd3702124899a37c23ded5";
+    sha256 = "1qmxm6pwwass2kpyg41nhkmyq0g74pyk517sq68dcgs6340ii7fs";
+  };
+
+  postPatch = ''
+    substituteInPlace common.pri \
+      --replace '$$[QT_INSTALL_HEADERS]/QtXmlPatterns' '${getDev qtxmlpatterns}/include/QtXmlPatterns' \
+      --replace '$$[QT_INSTALL_HEADERS]/QtSvg' '${getDev qtsvg}/include/QtSvg' \
+      --replace '$$[QT_INSTALL_HEADERS]/' '${getDev qtbase}/include/' \
+      --replace '$$[QT_INSTALL_HEADERS]' '${getDev qtbase}'
+    substituteInPlace src/app/tape/tape.pro \
+      --replace '$$[QT_INSTALL_BINS]/rcc' '${getDev qtbase}/bin/rcc'
+    substituteInPlace src/app/translations.pri \
+      --replace '$$[QT_INSTALL_BINS]/$$LRELEASE' '${getDev qttools}/bin/lrelease'
+    substituteInPlace src/app/valentina/mainwindowsnogui.cpp \
+      --replace 'define PDFTOPS "pdftops"' 'define PDFTOPS "${getBin poppler_utils}/bin/pdftops"'
+    substituteInPlace src/app/valentina/dialogs/dialogsavelayout.h \
+      --replace 'define PDFTOPS "pdftops"' 'define PDFTOPS "${getBin poppler_utils}/bin/pdftops"'
+  '';
+
+  nativeBuildInputs = [ qmake qttools ];
+
+  buildInputs = [ qtbase qtsvg qtxmlpatterns poppler_utils ];
+
+  configurePhase = ''
+    qmake PREFIX=/ Valentina.pro -r "CONFIG += noTests noRunPath no_ccache noDebugSymbols"
+  '';
+
+  installFlags = [ "INSTALL_ROOT=$(out)" ];
+
+  postInstall = ''
+    mv $out/usr/share $out/
+    rmdir $out/usr
+
+    mkdir -p $out/share/man/man1
+    gzip -9c dist/debian/valentina.1 > $out/share/man/man1/valentina.1.gz
+    gzip -9c dist/debian/tape.1 > $out/share/man/man1/tape.1.gz
+
+    mkdir -p $out/share/mime/packages
+    cp dist/debian/valentina.sharedmimeinfo $out/share/mime/packages/valentina.xml
+    cp dist/debian/valentina.mime $out/share/mime/packages/valentina
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "An open source sewing pattern drafting software";
+    homepage = https://valentinaproject.bitbucket.io/;
+    license = licenses.gpl3;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ jfrankenau ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17720,6 +17720,8 @@ with pkgs;
 
   utox = callPackage ../applications/networking/instant-messengers/utox { };
 
+  valentina = libsForQt5.callPackage ../applications/misc/valentina { };
+
   vanubi = callPackage ../applications/editors/vanubi { };
 
   vbindiff = callPackage ../applications/editors/vbindiff { };

